src/python/bot/tasks/minimize_task.py

```diff
@@ -1395,8 +1420,8 @@ def minimize_file(file_path,
 
   # Specialized minimization strategy for javascript.
   if file_path.endswith('.js'):
-    return do_js_minimization(test_function, get_temp_file, data, deadline,
-                              threads, cleanup_interval, delete_temp_files)
+    do_js_minimization(test_function, get_temp_file, data, deadline, threads,
```

Comment:
why remove the return ?

Reply:
Good catch, that was a mistake. When we update minimization in the next CL we should add more extensive unit tests. I don't think this code branch is tested at all right now.